src/styles/Teams.css

```diff
@@ -0,0 +1,38 @@
+.teams {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  padding: 0 6px;
+  gap: 0.5rem;
+}
+
```

Comment:
- [ ] Great work using grid to lay your elements on the page! 🎉 However, I've noticed that it doesn't quite match the provided design. 🧐 Please take a look at the screenshots below 📷 and make the necessary adjustments to align your layout with the design:

design project
![Screenshot from 2023-09-14 11-55-22](https://github.com/ngalamac/react-redux-capstone/assets/112261237/b93c1027-98bf-4396-91c0-7e08bfb018ef)

Your design project
![Screenshot from 2023-09-14 11-51-34](https://github.com/ngalamac/react-redux-capstone/assets/112261237/d537684a-c480-4332-88a0-88ade5b255a6)

You're doing a fantastic job! Keep up the good work! 👍😊